packages/graphql-playground-react/scripts/start.js

```diff
@@ -1,99 +1,99 @@
+const HOST = process.env.HOST || '0.0.0.0'
 
 // We attempt to use the default port but if it is busy, we offer the user to
 // run on a different port. `detect()` Promise resolves to the next free port.
 choosePort(HOST, DEFAULT_PORT)
-  .then(port => {
-  if (port == null) {
-  // We have not found a port.
-  return;
-}
-const protocol = process.env.HTTPS === 'true' ? 'https' : 'http';
-const appName = require(paths.appPackageJson).name;
-const urls = prepareUrls(protocol, HOST, port);
-// Create a webpack compiler that is configured with custom messages.
-const compiler = createCompiler({webpack, config, appName, urls, useYarn});
-// Load proxy config
-const proxySetting = require(paths.appPackageJson).proxy;
-const proxyConfig = prepareProxy(proxySetting, paths.appPublic);
-// Serve webpack assets generated by the compiler over a web sever.
-const serverConfig = createDevServerConfig(
-  proxyConfig,
-  urls.lanUrlForConfig
-);
-const devServer = new WebpackDevServer(compiler, serverConfig);
-// Launch WebpackDevServer.
-devServer.listen(port, HOST, err => {
-  if (err) {
-    return console.log(err);
-  }
-  if (isInteractive) {
-    clearConsole();
-  }
-  console.log(chalk.cyan('Starting the development server...\n'));
-openBrowser(urls.localUrlForBrowser);
-});
+  .then((port) => {
+    if (port == null) {
+      // We have not found a port.
+      return
+    }
+    const protocol = process.env.HTTPS === 'true' ? 'https' : 'http'
+    const appName = require(paths.appPackageJson).name
+    const urls = prepareUrls(protocol, HOST, port)
+    // Create a webpack compiler that is configured with custom messages.
+    const compiler = createCompiler({ webpack, config, appName, urls, useYarn })
+    // Load proxy config
+    const proxySetting = require(paths.appPackageJson).proxy
+    const proxyConfig = prepareProxy(proxySetting, paths.appPublic)
+    // Serve webpack assets generated by the compiler over a web sever.
+    const serverConfig = createDevServerConfig(
+      proxyConfig,
+      urls.lanUrlForConfig,
+    )
+    const devServer = new WebpackDevServer(compiler, serverConfig)
+    // Launch WebpackDevServer.
+    devServer.listen(port, HOST, (err) => {
+      if (err) {
+        return console.log(err)
+      }
+      if (isInteractive) {
+        clearConsole()
+      }
+      console.log(chalk.cyan('Starting the development server...\n'))
+      openBrowser(urls.localUrlForBrowser)
+    })
 
-['SIGINT', 'SIGTERM'].forEach(function(sig) {
-  process.on(sig, function() {
-    devServer.close();
-    process.exit();
-  });
-});
-})
-.catch(err => {
-  if (err && err.message) {
-  console.log(err.message);
-}
-process.exit(1);
-});
+    ;['SIGINT', 'SIGTERM'].forEach(function (sig) {
+      process.on(sig, function () {
+        devServer.close()
+        process.exit()
+      })
+    })
+  })
+  .catch((err) => {
+    if (err && err.message) {
+      console.log(err.message)
+    }
+    process.exit(1)
+  })
```

Comment:
changed this file so we can listen on 3005 while developing and not collide with GGT's port usage.

https://www.youtube.com/watch?v=tG35R8F2j8k